### ISSUE TITLE
Upgrading Bonfire 1 to Bonfire 2

### ISSUE THREAD

**Issue by Kaushali-ITSthe1** [lonnieezell/Bonfire2#364]
I have an application developed by using Bonfire 1. The PHP version is 5.6. Now I want to upgrade same modules compatible with latest, PHP 7.4+.How to upgrade bonfire 1 application to Bonfire 2.

**Comment by cijagani** [comment]
As per my knowledge, it seems very complicated. Cz bonfire 1 was based on ci 3 and bonfire 2 is based on ci4. Moreover bonfire is still beta version

**Comment by dgvirtual** [comment]
I guess this could be turned into a short section in the bonfire documentation (perhaps just explaining that there is no easy way to convert), and the issue should be closed. Is the assessment about no easy conversion way accurate?

**Comment by lonnieezell** [comment]
Correct. There is no direct upgrade path. It is a completely new project based on rewritten framework.
